Soomla/CCProfileConsts.cpp

```diff
@@ -89,17 +91,19 @@ namespace soomla {
     char const *CCProfileConsts::EVENT_INVITE_FAILED = "com.soomla.profile.events.social.InviteFailedEvent";
     char const *CCProfileConsts::EVENT_INVITE_CANCELLED = "com.soomla.profile.events.social.InviteFailedEvent";
 
-    char const *CCProfileConsts::EVENT_UP_GET_LEADERBOARDS_STARTED = "com.soomla.profile.events.social.GetLeaderboardsStartedEvent";
-    char const *CCProfileConsts::EVENT_UP_GET_LEADERBOARDS_FINISHED = "com.soomla.profile.events.social.GetLeaderboardsFinishedEvent";
-    char const *CCProfileConsts::EVENT_UP_GET_LEADERBOARDS_FAILED = "com.soomla.profile.events.social.GetLeaderboardsFailedEvent";
+    char const *CCProfileConsts::EVENT_GET_LEADERBOARDS_STARTED = "com.soomla.profile.events.social.GetLeaderboardsStartedEvent";
+    char const *CCProfileConsts::EVENT_GET_LEADERBOARDS_FINISHED = "com.soomla.profile.events.social.GetLeaderboardsFinishedEvent";
+    char const *CCProfileConsts::EVENT_GET_LEADERBOARDS_FAILED = "com.soomla.profile.events.social.GetLeaderboardsFailedEvent";
 
```

Comment:
@vedi, all literals above doesn't have this prefix. I did it to unify coding style.